build.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+const fs = require('fs');
+const exec = require('child_process').execSync;
+
+const DEST = 'keepassxc-browser';
+const DEFAULT = 'manifest_default.json';
+const BROWSERS = {
+    'Firefox': 'manifest_firefox.json',
+    'Chromium': 'manifest_chromium.json',
+};
+
+function adjustManifest(manifest) {
+    const manifestFile = fs.readFileSync(DEFAULT, 'utf8');
+    const data = JSON.parse(manifestFile);
+    const browser = manifest.substring(manifest.indexOf('_') + 1, manifest.indexOf('.'));
+
+    if (manifest.includes('firefox')) {
+        for (const elem in data['icons']) {
+            data['icons'][elem] = 'icons/keepassxc.svg';
+        }
+        for (const elem in data['browser_action']['default_icon']) {
+            data['browser_action']['default_icon'][elem] = 'icons/keepassxc.svg';
+        }
+    } else if (manifest.includes('chromium')) {
+        delete data['applications'];
+    }
+
+    fs.writeFileSync(manifest, JSON.stringify(data, null, 4));
+    return `keepassxc-browser_${data['version']}_${browser}.zip`;
+};
+
+fs.copyFileSync(`${DEST}/manifest.json`, `./${DEFAULT}`);
+
+for (const browser in BROWSERS) {
+    console.log(`KeePassXC-Browser: Creating extension package for ${browser}`);
+    const fileName = adjustManifest(BROWSERS[browser]);
+    fs.copyFileSync(BROWSERS[browser], `${DEST}/manifest.json`);
+    exec(`rm -rf ${fileName}`);
+    exec(`zip -r -FS ${fileName} ${DEST}/* > /dev/null`);
+    exec(`rm -rf ${BROWSERS[browser]}`);
```

Comment:
This code is Linux/Mac-specific and doesn't work on Windows. To make it cross platform I suggest to replace `rm  -rf` with [fs-extra](https://github.com/jprichardson/node-fs-extra/blob/HEAD/docs/remove-sync.md) and `zip` with [archiver](https://www.npmjs.com/package/archiver)

Reply:
Good point.